Src/Presentation/OpenChat.API/Controllers/UsersController.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+
+namespace OpenChat.API.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class UsersController : ControllerBase
+    {
+        public Task Post() {
+            throw new Exception();
```

Comment:
I think is a good practice to throw NotImplementedException at this stage.

Reply:
> I think is a good practice to throw NotImplementedException at this stage.

Yes. This is what I wanted actually. :)